### DISCUSSION TITLE
很奇怪，在游戏崩溃之后，启动器会消失不见

### DISCUSSION THREAD

**Discussion by RZZCCCP** [Meloong-Git/PCL#2322] [提问] [answered]
如标题所示，我的是赞助版。在游戏当中模组冲突，亦或者是其他的原因导致游戏闪退。pcl启动器会莫名其妙消失（在垃圾桶里面也看不到）就像我根本没下过这个软件一样，除了启动器消失不见以外，其他的文件都正常存在

**Comment by zkitefly** [comment]
#666 可能是被杀软等杀毒软件删除了，检查下？

**Comment by WForst-Breeze** [comment, marked as answer]
可以在最新快照版本内置帮助库中找到可能的解决办法（将 PCL 添加到 杀软&WD 排除项），依步骤操作完成后再次检查问题是否依然存在。
